### PR TITLE
Change name of the app.

### PR DESCRIPTION
**Pull request by SjoenH**
Changed the name of the app
Added icon-asset for ios + android

### CODE REVIEW

File: android/app/src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="no.kodekameratene.jonah">
+    package="com.example.oase">
```
Comment (SjoenH) [inline android/app/src/debug/AndroidManifest.xml]: Should be no.oase.festival

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="no.kodekameratene.jonah">
+    package="com.example.oase">
```
Comment (SjoenH) [inline android/app/src/main/AndroidManifest.xml]: no.oase.festival

File: android/app/src/main/java/com/example/oase/MainActivity.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package no.kodekameratene.jonah;
+package com.example.oase;
```
Comment (SjoenH) [inline android/app/src/main/java/com/example/oase/MainActivity.java]: no.oase.festival

File: android/app/src/profile/AndroidManifest.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="no.kodekameratene.jonah">
+    package="com.example.oase">
```
Comment (SjoenH) [inline android/app/src/profile/AndroidManifest.xml]: yea, you know it

**Review by marose14 — APPROVED** [review]
Awesome!

**Pull request merged**